samples/uppercase/README.adoc

```diff
@@ -26,14 +26,14 @@ N.B. this sample uses Spring (without really needing to, but just to show how it
 === create the function and its input topic
 
 ```
-riff create java --input words --handler 'uppercase&main=functions.Application'
+riff create java --handler 'uppercase&main=functions.UppercaseApplication'
```

Comment:
the nice thing about using `--input words` is that this sample and `wordcount` could run side-by-side as a demo for one-to-many stream to function (also with one of those functions being item-at-a-time and the other being a streaming example), but I guess that's always an option even if not in the README (I will just have to change my habit of copy/paste-ing from the README when I run that demo ;)

Reply:
if you do `delete --all` on wordcount you can't do `delete --all` on uppercase too since the topic is gone that command will fail

Reply:
so for these standalone samples I think we should avoid sharing topics